cf/test/test_weights.py

```diff
@@ -291,6 +291,43 @@ def test_weights_line_area_ugrid(self):
         )
         self.assertEqual(w.Units, cf.Units("m"))
 
+    def test_weights_cell_measures_coordinates(self):
+        import cf
```

Comment:
```suggestion
```

Since it is already imported. (With removal of the next line down, the 'suggestion' feature won't let me include that.)